src/map/status.c

```diff
@@ -248,7 +248,7 @@ static void initChangeTables(void)
 	status->set_sc( SM_AUTOBERSERK       , SC_AUTOBERSERK  , SCB_NONE );
 	add_sc( TF_SPRINKLESAND      , SC_BLIND           );
 	add_sc( TF_THROWSTONE        , SC_STUN            );
-	status->set_sc( MC_LOUD              , SC_SHOUT        , SCB_STR );
+	status->set_sc( MC_LOUD              , SC_SHOUT        , SCB_STR ); // i made a mistake here, aparently SCB_WATAK don`t need to be pass
```

Comment:
this comment is confusing, I didn't get it

Reply:
i THINK on my first implementation i set SCB_STR|SBC_WATK but realize that isn't needed for the atk bonus

Reply:
After I applied the diff I got an error where the +ATK wouldn't count unless I unequiped and equiped the weapon. Therefore I thinks is better to set the SCB_* here.
I also have changed the SCB_WATK to SCB_BATK and now its working fine. So the new line should be:

status->set_sc( MC_LOUD              , SC_SHOUT        , SCB_STR | SCB_BATK );